packages/button/src/Button.ts

```diff
@@ -136,4 +177,15 @@ export class Button extends SizedMixin(ButtonBase) {
             this.setAttribute('variant', this.variant);
         }
     }
+
+    protected override renderButton(): TemplateResult {
+        return html`
+            ${this.buttonContent}
+            <sp-progress-circle
+                indeterminate
+                static="white"
+                size="s"
+            ></sp-progress-circle>
```

Comment:
Should this always be available in the doc? If that's the approach we want, then we'll need something like `aria-hidden=${ifDefined(!this.pending ? 'true' : undefined)}` to keep it off of the accessibility tree.

Also, it doesn't look like this registration is ever loaded. Should it be static, dynamic, in the docs?

Reply:
yeah my feeling is it should be in the docs only (you don't want to lazy load this it might not show up for long as you're only supposed to use these pending states for < 5s loads). Maybe a dev mode warning if its used and pending-circle isn't registered?

Reply:
Modified this to only add the element when needed, this avoids the aria issue. Have still not made a decision on importing, I'm leaning towards just documentation of the need for `sp-progress-circle` being a required dependency when using `pending` state though? This would then avoid unnecessary payload. Maybe add a `dev` warning if the element is not registered?